doc/source/reference/commands/io_flush.rst

```diff
@@ -279,6 +279,81 @@ isn't specified:
 .. include:: ../../example/reference/commands/io_flush/recursive_default.log
 .. io_flush
 
+Since 9.0.2, ``recursive`` behavior is extended to flush not only target object
+and child objects, but also related objects. The related objects are:
+
+* A referenced table
+* A related index column (There is source column in target TABLE_NAME)
+* A table of related index column (There is source column in target TABLE_NAME)
+
+It is useful not to forget flushing related objects.
+
+For example, ``--recursive yes`` is specified for ``TABLE_NAME``, this
+option executes equivalent to the following commands internally.
+
+- Flush table and its columns::
+
+    io_flush --target_name TABLE_NAME
```

Comment:
a default value of `recursive ` is `yes`.
Therefore, I think if we execute this command, flush not only target object and child objects but also related objects.

Reply:
I've missed it, so fixed it.